code/modules/clothing/suits/armor.dm

```diff
@@ -41,6 +40,10 @@
 /obj/item/clothing/suit/armor/vest/security
 	name = "security armor"
 	desc = "An armored vest that protects against some damage. This one has a clip for a holobadge."
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/species/vox/suit.dmi',
```

Comment:
no need for vox in the list here in `obj/item/clothing/suit/armor/vest/security` , since its already in `/obj/item/clothing/suit/armor`.